config/deploy.rb

```diff
@@ -91,3 +91,42 @@
     end
   end
 end
+
+namespace :env do
+  desc 'Set a Figgy environment variable'
+  task :set do |_task, args|
+    on roles(:all) do
+      abort "Environment variables and values must be specified. `env:set['ENV_VAR=value']`" if args.extras.empty?
+      config_file = '/home/deploy/app_configs/figgy'
```

Comment:
Do we do this somewhere else? This makes me kind of uncomfortable, since it'll get wiped out on a provision.

Reply:
Good point.

I pulled it from OL. The code is written to not require this environment variable at all, and it will default to not being in read-only mode. So the potentially bad scenario would be needing to provision during a read-only downtime.

That's not what we're up to this weekend, so maybe it's okay for now?

Also happy to hear other ideas.

Reply:
For the sake of finishing reindexing, I'm fine with this for now, but I'd like an issue at least to manage the switch in the database somewhere.

Reply:
here's an issue, feel free to add comments: https://github.com/pulibrary/figgy/issues/2872